### PR TITLE
Corrections from PR 268 and addition of experiment script and notebook

### PR DESCRIPTION
**Pull request by FishAres**
run_dendrite_experiment.py -> will run a couple of sequential training experiments
dend_layer_playground.ipynb -> notebook for experimentation

### CODE REVIEW

File: nupic/research/frameworks/continuous_learning/dend_kwinners.py [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+# ----------------------------------------------------------------------
+
+import abc
+
+import numpy as np
+import torch
+import torch.nn as nn
+
+
+class DKWinnersBase(nn.Module, metaclass=abc.ABCMeta):
```
Comment (subutai) [inline nupic/research/frameworks/continuous_learning/dend_kwinners.py:29]: This file overlaps with dc_k_winners.py - it has many of the same classes. What's the difference?

File: nupic/research/frameworks/continuous_learning/dendrite_layers.py [hunk 2]
```diff
@@ -124,51 +127,72 @@ class DendriteLayer(nn.Module):
...
-        )
-        self.act_fun.k = 1
-        self.act_fun.k_inference = 1
+        if self.act_fun_type == "kwinner":
+            self.act_fun = DendriteKWinners2d(
+                channels=self.out_dim,
+                k=1,
+                local=True,
+            )
+            self.act_fun.k = 1
```
Comment (subutai) [inline nupic/research/frameworks/continuous_learning/dendrite_layers.py:159]: Why set k to 1 right after are initializing it with k=1?

File: nupic/research/frameworks/continuous_learning/dendrite_layers.py [hunk 3]
```diff
@@ -124,51 +127,72 @@ class DendriteLayer(nn.Module):
...
+        else:
+            self.act_fun = torch.sigmoid
 
-    def forward(self, x):
+    def forward(self, x, y=None):
```
Comment (subutai) [inline nupic/research/frameworks/continuous_learning/dendrite_layers.py:163]: Not clear what this is doing. What does y represent? (put in docstring)

File: projects/continuous_learning/sync_ec2_file.sh [hunk 4]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```
Comment (subutai) [inline projects/continuous_learning/sync_ec2_file.sh:1]: Infrastructure stuff shouldn't be in this rep. In addition, the stuff here is specific to afisher. Please remove this file from here.

File: projects/continuous_learning/sync_ec2_repo.sh [hunk 5]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```
Comment (subutai) [inline projects/continuous_learning/sync_ec2_repo.sh:1]: Infrastructure stuff shouldn't be in this rep. In addition, the stuff here is specific to afisher. Please remove this file from here.

File: nupic/research/frameworks/continuous_learning/dendrite_layers.py (hunk 2 above)
Comment (FishAres) [reply nupic/research/frameworks/continuous_learning/dendrite_layers.py:159]: serial editing brain.. will remove

File: nupic/research/frameworks/continuous_learning/dend_kwinners.py (hunk 1 above)
Comment (FishAres) [reply nupic/research/frameworks/continuous_learning/dend_kwinners.py:29]: this is for the k=1 k-winner implementation. dc_k_winners.py is to control when the duty cycles are updated. i'll make sure to add a docstring and resolve ambiguity between similar class names

**Pull request merged**